result_report.py

```diff
@@ -14,9 +14,33 @@ def visit_test(self, test):
         elif test.status == 'PASS':
             self.passed_tests.append(test.name)
 
+    def add_component_version_table(self, file):
+        if len(endpoints) == 0:
+            return file
```

Comment:
I think we can just return None here.
```suggestion
            return None
```

Reply:
Yes